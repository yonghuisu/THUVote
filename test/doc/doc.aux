\relax 
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\@writefile{toc}{\contentsline {section}{\numberline {1}介绍}{1}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}设计}{2}{section.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}前端}{2}{subsection.2.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}后端}{2}{subsection.2.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.3}数据库}{2}{subsection.2.3}}
\@writefile{toc}{\contentsline {section}{\numberline {3}实现}{2}{section.3}}
\@writefile{toc}{\contentsline {section}{\numberline {4}预算}{2}{section.4}}
