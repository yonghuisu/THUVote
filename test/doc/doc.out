\BOOKMARK [1][-]{section.1}{介绍}{}% 1
\BOOKMARK [1][-]{section.2}{设计}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{前端}{section.2}% 3
\BOOKMARK [2][-]{subsection.2.2}{后端}{section.2}% 4
\BOOKMARK [2][-]{subsection.2.3}{数据库}{section.2}% 5
\BOOKMARK [1][-]{section.3}{实现}{}% 6
\BOOKMARK [1][-]{section.4}{预算}{}% 7
